src/js/modules/accounts/components/ProfilePage.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { Link } from "react-router-dom";
+import injectSheet from "react-jss";
+
+import { SignOutForm } from "./forms";
+import { signOut } from "../actions";
+
+const styles = {
+  successMessage: {
+    color: "green",
+  },
+};
+
+const ProfilePage = ({ classes, session, signOut, status, users }) => {
+  if (status.form === "signOut" && status.errors.length === 0) {
+    return (
+      <div>
+        <p className={classes.successMessage}>{status.message}</p>
+        <Link to="/myaccount">Sign in</Link>
+        &nbsp;or go back to the&nbsp;
+        <Link to="/">home page</Link>.
+      </div>
+    );
+  }
+  // The last if statement includes the following condition. This next if block
+  // is a message for those who directly visit myaccount/profile.
+  if (!session.userId) {
+    return (
+      <p>
+        You are not signed in. <Link to="/myaccount">Sign in</Link> or go back
+        to the <Link to="/">home page</Link>
+      </p>
+    );
+  }
+
+  const user = users[session.userId]
```

Comment:
Use a selector here. getCurrentUser selector

Reply:
Going to hold off on doing that. In most forms/cases, I still need the entire session to sign out (because I need all headers).